### ISSUE TITLE
Select multiple requests

### ISSUE THREAD

**Issue by jaikishantulswani** [Quitten/Autorize#42]
@Quitten it would be better if autorize have feature to select multiple request.

**Comment by Quitten** [comment]
Hey, can you please elaborate? 
batching multiple requests was merged at commit 53712ebb1c19dc6b4a94db50f736db3e37ddece1 thanks to @Regala.
Make sure you are working on latest release:
https://github.com/Quitten/Autorize/releases/tag/0.21

btw, I'm part of Wix security team and I think you are our best hacker1 researcher so keep on your great job and happy to know you are using my tools to do so :)

Please close the issue in case the feature implemented

**Comment by jaikishantulswani** [comment]
HI @Quitten Thank you for your kind words. This is the best I ever got as a compliment. 
I am using the latest release and in that I am not able to select multiple request with `ctrl`. Can you please let me know how I select multiple requests at once and send them to repeater.

Regards,
@jaikishantulswani

**Comment by Quitten** [comment]
Hey @jaikishantulswani, currently we didn't implement multiple selection on the JTable. I thought you were talking about the opposite direction.
Hope I will find time to implement it soon (or anyone else in the community will)

**Comment by jaikishantulswani** [comment]
@Quitten Hi,
Waiting for a feature where we are able to select multiple urls to retest them at once.

Regards,
@jaikishantulswani

**Comment by Quitten** [comment]
@jaikishantulswani - I am sorry but I cannot find time to do so soon.
Feel free to try to implement it yourself and ill be happy to merge it.
If you can't hope anyone else will contribute and implement it :)

**Comment by MrR10T** [comment]
@Quitten Yes, that's what I needed. Btw I will try myself to implement this, but I'm not sure I have the knowledge. I know some python but I don't use it a lot. I will definitly do a try.

**Comment by terminalJunki3** [comment]
@jaikishantulswani I’ve implemented multiple row select and it has been merged PR #105.  The way the retest functions currently work are not compatible yet and will have to be modified to take advantage of this new capability. Right now, the calls look at the row in focus which won’t work. So far I’ve only been able to connect delete functionality to the new multi row select feature allowing you to remove multiple rows at once.  @Quitten I’ve begun looking at converting over these functions but it’s a bit tricky. @jaikishantulswani or @Quitten if you have any insight into a good way of doing this please let me know and it could save me some time. That said, worst case scenario is I think I can have it done by the end of February if not before. Going to have to have multiple google tabs open for this one. 😀

**Comment by jaikishantulswani** [comment]
@terminalJunki3 Looks great, thanks for your contribution as it was needed from a long time

**Comment by terminalJunki3** [comment]
@jaikishantulswani Absolutely!
